src/views/Layout.jsx

```diff
@@ -15,6 +15,23 @@ import {
 import logo from '../images/PredictaCartLogo.png';
 
 export function Layout() {
+	const location = useLocation();
```

Comment:
`useLocation`: This is a hook provided by React Router, which returns the current location object that represents the current URL. You can think of it as a way of programmatically accessing the current browser URL's details.